website/docs/backends/types/s3.html.md

```diff
@@ -15,6 +15,7 @@ Stores the state as a given key in a given bucket on
 This backend also supports state locking and consistency checking via
 [Dynamo DB](https://aws.amazon.com/dynamodb/), which can be enabled by setting
 the `dynamodb_table` field to an existing DynamoDB table name.
+A single DynamoDB table can be used to lock multiple remote state files in S3 buckets by creating separate rows and prefixing the lock key with each bucket name. 
```

Comment:
"by creating separate rows and prefixing the lock key with each bucket name" is ambiguous: it could be interpreted as meaning that a user of Terraform needs to take care to somehow prefix the lock key with each bucket name, when in fact it's something that is handled automatically by Terraform.

Suggested changed:

> A single DynamoDB table can be used to lock multiple remote state files. Terraform generates key names that include the values of the `bucket` and `key` variables.

Reply:
Agreed! @charlottemach, would you be willing to merge @jfirebaugh's suggestion? Once you do that, I'm willing to merge.

```suggestion
A single DynamoDB table can be used to lock multiple remote state files. Terraform generates key names that include the values of the `bucket` and `key` variables.
```